apps/heft/includes/jest-shared.config.json

```diff
@@ -3,9 +3,11 @@
 
   "rootDir": "../../../../",
   "testURL": "http://localhost/",
-  "testMatch": ["<rootDir>/src/**/*.test.ts?(x)"],
+  "testMatch": ["<rootDir>/src/**/*.test.{ts,tsx}"],
+  "transformIgnorePatterns": [],
   "transform": {
-    "src[\\\\/].+\\.ts$": "<rootDir>/node_modules/@rushstack/heft/lib/plugins/JestPlugin/HeftJestSrcTransform.js"
+    "\\.(ts|tsx)$": "<rootDir>/node_modules/@rushstack/heft/lib/exports/jest-build-transform",
+    "\\.(css|jpg|png|gif|mp3)$": "<rootDir>/node_modules/@rushstack/heft/lib/exports/jest-identity-mock-transform"
```

Comment:
It's probably valuable to handle CSS differently. Modules expected to be handled by webpack's `file-loader` should be represented as strings, but CSS should probably be represented as a `{ [cssClassName: string]: string }` object.

Reply:
@iclanton We chatted about this and decided to handle it in a separate PR